lib/generators/stimulus_reflex/templates/config/importmap.rb.tt

```diff
@@ -0,0 +1,17 @@
+# Pin npm packages by running ./bin/importmap
+
+pin "application", preload: true
+pin "@hotwired/turbo-rails", to: "turbo.min.js"
+pin "@hotwired/stimulus", to: "stimulus.min.js"
+pin "@hotwired/stimulus-loading", to: "stimulus-loading.js"
+
+pin "@rails/actioncable", to: "actioncable.esm.js"
+pin "cable_ready", to: "https://devbuilds.herokuapp.com/package/npm/cable_ready/latest"
+pin "stimulus_reflex", to: "https://devbuilds.herokuapp.com/package/npm/stimulus_reflex/latest"
+pin "morphdom", to: "https://ga.jspm.io/npm:morphdom@2.6.1/dist/morphdom.js"
+pin "mrujs", to: "https://ga.jspm.io/npm:mrujs@0.10.1/dist/index.module.js"
+pin "mrujs/plugins", to: "https://ga.jspm.io/npm:mrujs@0.10.1/plugins/dist/plugins.module.js"
```

Comment:
This needs to be dynamic based on the user-selection, if we actullay want to include Mrujs in the installer.
```suggestion
```

Reply:
Yes! Good catch; it actually *was* [right here](https://github.com/stimulusreflex/stimulus_reflex/pull/610/files#diff-fc21c7099580cece0a61b9d911dbc367a687f4384f621ab6e1512aaae1ae9a11R14) but I still have not been able to figure out how to require the `importmap-rails` actions into these scripts so that we can call `pin` etc inside of the mrujs task.

If you can help crack that code, I'll move the pins back to the mrujs step. (And that still doesn't mean we'll go the mrujs route! I just want to make sure the mrujs path can be evaluated fairly.)